citesphere/src/main/java/edu/asu/diging/citesphere/core/authority/impl/ConceptpowerImporter.java

```
@@ -0,0 +1,156 @@
+        restTemplate = new RestTemplate();
+    }
+
+    /*
+     * (non-Javadoc)
+     * 
+     * @see
+     * edu.asu.diging.citesphere.authority.impl.AuthorityImporter#isResponsible(java
+     * .lang.String)
+     */
+    @Override
+    public boolean isResponsible(String source) {
+
+        if (CONCEPTPOWER.equals(source)) {
+            return true;
+        }
+
+        return false;
+    }
+
+    /*
+     * (non-Javadoc)
+     * 
+     * @see edu.asu.diging.citesphere.authority.impl.AuthorityImporter#
+     * retrieveAuthorityData(java.lang.String)
+     */
+    @Override
+    public IImportedAuthority retrieveAuthorityData(String searchString)
+            throws URISyntaxException, AuthorityServiceConnectionException {
+        return null;
+    }
+
+    @Override
+    public String getId() {
+        return ID;
+    }
+
+    @Override
+    public AuthoritySearchResult searchAuthorities(String firstName, String lastName, int page, int pageSize)
+            throws URISyntaxException, AuthorityServiceConnectionException {
+
+        HttpComponentsClientHttpRequestFactory factory = new HttpComponentsClientHttpRequestFactory();
+        HttpClient httpClient = HttpClientBuilder.create().setRedirectStrategy(new LaxRedirectStrategy()).build();
+        factory.setHttpClient(httpClient);
+        restTemplate.setRequestFactory(factory);
+
+        RequestEntity<Void> request;
+        try {
+
+            String url = conceptpowerURL + conceptpowerSearchKeyword + URLEncoder
+                    .encode(this.getConceptpowerSearchString(firstName, lastName), StandardCharsets.UTF_8.toString())
+                    + "&page=" + page;
+            URI uri = UriComponentsBuilder.fromUriString(url.toString()).build(true).toUri();
+
+            request = RequestEntity.get(uri).accept(MediaType.APPLICATION_JSON).build();
+        } catch (UnsupportedEncodingException e) {
+            throw new AssertionError(StandardCharsets.UTF_8.toString() + " is not supported");
```

don't swallow the exception